crates/cli/src/args.rs

```diff
@@ -195,6 +195,10 @@ pub struct Args {
     /// Avoid saving a summary report
     #[arg(long, help_heading = "Output Options")]
     pub no_report: bool,
+    
+    /// Skip if writing if dataframe is empty
```

Comment:
default behavior should be to write the empty files because thats what allows efficient idempotency when you run the same command multiple times

so would be good to rename this as `--skip-empty`

Reply:
True, I've renamed to --skip-empty, which will write empty dfs by default.